### PR TITLE
Add oggbundle pipeline and example data imports.

### PR DESCRIPTION
**Pull request by deiferni**
First implementation of the pipeline to import a bundle. It is based on our already customised `ftw.inflator` pipeline `opengever.setup.content`.

### CODE REVIEW

File: opengever/setup/tests/assets/oggbundle/dossiers.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+[
+  {
+    "guid": "4ed93561c4004c668feb8124b55897fa",
+    "parent_guid": "aed9f3db7b3b4713b61a7d7295362ea4",
+    "comments": "Vreni Meier ist ein Tausendsassa",
+    "keywords": [],
+    "reference_number": "1",
```
Comment (lukasgraf) [inline opengever/setup/tests/assets/oggbundle/dossiers.json]: Why `reference_number` and not `reference_number_prefix`?

Comment (deiferni) [inline opengever/setup/tests/assets/oggbundle/dossiers.json]: that's what the schema definition requires 😳

Comment (lukasgraf) [inline opengever/setup/tests/assets/oggbundle/dossiers.json]: Never mind, I was thinking of repository folders 🤕

**Comment by deiferni** [comment]
@lukasgraf updated

**Pull request merged**